tests/phpunit/tests/dependencies/scripts.php

```diff
@@ -2931,6 +2931,21 @@ static function() {
 		$this->assertSame( $expected, $print_scripts );
 	}
 
+	/**
+	 * Ensure tinymce scripts aren't loading async.
+	 *
+	 * @ticket 58648
+	 */
+	public function test_printing_tinymce_scripts() {
+		global $wp_scripts;
+
+		wp_register_tinymce_scripts( $wp_scripts, true );
+
+		$actual = get_echo( 'wp_print_scripts', array( array( 'wp-tinymce' ) ) );
+
+		$this->assertStringNotContainsString( 'async', $actual );
```

Comment:
For good measure perhaps:

```suggestion
		$this->assertStringNotContainsString( 'async', $actual );
		$this->assertStringNotContainsString( 'defer', $actual );
```

Reply:
If we add the the suggested, reminder to add messages.

Reply:
Handled in [3d1a245](https://github.com/WordPress/wordpress-develop/pull/4852/commits/3d1a2457cabaa2cb15abf26a2a22ebe4d539d6a8).